lib/landable/engine.rb

```diff
@@ -65,5 +65,11 @@ class Engine < ::Rails::Engine
         # end
       end
     end
+
+    initializer 'landable.enable_hutch' do |app|
```

Comment:
Unused block argument - `app`. You can omit the argument if you don't care about it.